README.md

```diff
@@ -108,14 +239,41 @@ If the argument you provide when calling the function holds values [a,b,c...z],
 Make sure to handle two special cases: length 0 (then return _true_) and length 1 (then return that single value).
 Again, don't use &&.
 
+Solution:
+
+```
+  function andN(values){
+  for (i = 0; i <= values.length; i++){
+    if (values[i] === false) {
+      return false;
+    } };
+    if (i = values.length + 2) {
+      return true;
+    }
+    ;}
+  ```
+
 **d)**
 You've just realized that your effort was doomed: neither of your functions can replace the && operator in certain circumstances.  Explain why, and find an example which demonstrates failure.
 
+
+Examples of failures and why:
+
+a. 
+```
+and2(@, -)
```

Comment:
4d) The big difference with `&&` and `||` is that they are lazy expressions: they will only evaluate as far as they need to. So, assuming you don't have a variable called "cats", `false && cats` will simply return `false`, whilie `and2(false, cats)` will give you an uncaught reference error.